deploy/operator/libvirt_disks.sh

```diff
@@ -0,0 +1,45 @@
+if [ -z "${NODES}" ]; then
+    export NODES=$(virsh list --name | grep worker || virsh list --name | grep master)
+fi
+
+if [ -z "${DISKS}" ]; then
+    echo "You must provide DISKS env-var. For example:"
+    echo "    DISKS=\$(echo sd{b..f}) bash ./libvirt_disks.sh create"
+    exit 1
+fi
+
+function create() {
+    export SIZE=${SIZE:-50G}
+
+    echo "Creating libvirt disks and attaching them..."
+    for node in ${NODES}; do
+        for disk in ${DISKS}; do
+            qemu-img create -f raw "/tmp/${node}-${disk}.img" "${SIZE}"
+            virsh attach-disk "${node}" "/tmp/${node}-${disk}.img" "${disk}"
```

Comment:
Verify that the file doesn't exist
```suggestion
            img_path="/tmp/${node}-${disk}.img"
            if [ ! -f ${img_path} ]; then
                qemu-img create -f raw ${img_path} ${SIZE}
                virsh attach-disk "${node}" ${img_path} "${disk}"
            fi
```